MakerTracker/ClientApp/src/app/Profiles/init-profile/init-profile.component.html

```diff
@@ -1,290 +1,288 @@
 <h2 class="flex-shrink">Welcome to Maker Tracker</h2>
```

Comment:
99% of this file's changes are due to indentation. I added a parent `<mat-dialog-content>` tag.